core/src/OC/password-confirmation.js

```diff
@@ -22,110 +22,23 @@
  *
  */
 
-import _ from 'underscore'
-import $ from 'jquery'
-import moment from 'moment'
-import { generateUrl } from '@nextcloud/router'
-
-import OC from './index.js'
+import { confirmPassword, isPasswordConfirmationRequired } from '@nextcloud/password-confirmation'
```

Comment:
We also need to import styles here.

It works because they were already imported from another place, but we cannot count on that:
- May break if another place changes
- May break on chunks separating

```suggestion
import { confirmPassword, isPasswordConfirmationRequired } from '@nextcloud/password-confirmation'
import '@nextcloud/password-confirmation/style.css'
```